docs/redoc-vendor-extensions.md

```diff
@@ -2,56 +2,56 @@
 
 You can use the following [vendor extensions](https://swagger.io/specification/#specificationExtensions) with Redoc.
 
-<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
-
-- [ReDoc vendor extensions](#redoc-vendor-extensions)
-		- [Swagger Object](#swagger-object)
-			- [x-servers](#x-servers)
-			- [x-tagGroups](#x-taggroups)
-					- [How to use with Redoc](#how-to-use-with-redoc)
-			- [<a name="tagGroupObject"></a>Tag Group Object](#a-nametaggroupobjectatag-group-object)
-					- [Fixed fields](#fixed-fields)
-					- [x-tagGroups example](#x-taggroups-example)
-			- [x-ignoredHeaderParameters](#x-ignoredheaderparameters)
-					- [How to use with Redoc](#how-to-use-with-redoc)
-					- [x-ignoredHeaderParameters example](#x-ignoredheaderparameters-example)
-		- [Info Object](#info-object)
-			- [x-logo](#x-logo)
-					- [How to use with Redoc](#how-to-use-with-redoc)
-			- [<a name="logoObject"></a>Logo Object](#a-namelogoobjectalogo-object)
-					- [Fixed fields](#fixed-fields)
-					- [x-logo example](#x-logo-example)
-		- [Tag Object](#tag-object)
-			- [x-traitTag](#x-traittag)
-					- [How to use with Redoc](#how-to-use-with-redoc)
-					- [x-traitTag example](#x-traittag-example)
-			- [x-displayName](#x-displayname)
-		- [Operation Object vendor extensions](#operation-object-vendor-extensions)
-			- [x-codeSamples](#x-codesamples)
-					- [How to use with ReDoc](#how-to-use-with-redoc)
-			- [<a name="codeSampleObject"></a>Code Sample Object](#a-namecodesampleobjectacode-sample-object)
-					- [Fixed fields](#fixed-fields)
-					- [Code Sample Object example](#code-sample-object-example)
-		- [Parameter Object](#parameter-object)
-			- [x-examples](#x-examples)
-					- [How to use with ReDoc](#how-to-use-with-redoc)
-		- [Response Object vendor extensions](#response-object-vendor-extensions)
-			- [x-summary](#x-summary)
-					- [Usage in ReDoc](#usage-in-redoc)
-		- [Schema Object](#schema-object)
-			- [x-nullable](#x-nullable)
-					- [How to use with ReDoc](#how-to-use-with-redoc)
-			- [x-extendedDiscriminator](#x-extendeddiscriminator)
-					- [How to use with ReDoc](#how-to-use-with-redoc)
-					- [x-extendedDiscriminator example](#x-extendeddiscriminator-example)
-			- [x-additionalPropertiesName](#x-additionalpropertiesname)
-					- [How to use with ReDoc](#how-to-use-with-redoc)
-					- [x-additionalPropertiesName example](#x-additionalpropertiesname-example)
-			- [x-explicitMappingOnly](#x-explicitmappingonly)
-					- [How to use with ReDoc](#how-to-use-with-redoc)
-					- [x-explicitMappingOnly example](#x-explicitmappingonly-example)
-
-<!-- /TOC -->
+<!-- MDTOC maxdepth:6 firsth1:1 numbering:0 flatten:0 bullets:1 updateOnSave:1 -->
+
+- [ReDoc vendor extensions](#redoc-vendor-extensions)   
+      - [Swagger Object](#swagger-object)   
+         - [x-servers](#x-servers)   
+         - [x-tagGroups](#x-taggroups)   
+               - [How to use with Redoc](#how-to-use-with-redoc)   
```

Comment:
For some reason these aren't indented further than the one above it (even though it looks like they should be).